torch/optim/nadam.py

```diff
@@ -119,8 +121,10 @@ def step(self, closure=None):
             &\rule{110mm}{0.4pt}                                                                 \\
             &\textbf{for} \: t=1 \: \textbf{to} \: \ldots \: \textbf{do}                         \\
             &\hspace{5mm}g_t           \leftarrow   \nabla_{\theta} f_t (\theta_{t-1})           \\
-            &\hspace{5mm}if \: \lambda \neq 0                                                    \\
+            &\hspace{5mm}\textbf{if} \: \lambda \neq 0 \text{ and not } \textit{decoupled_weight_decay} \\
```

Comment:
Should mention decoupled_weight_decay as an input!

Reply:
Thanks for noticing, done!